### ISSUE TITLE
glide.lock refers to spew, but it's not used.

### ISSUE THREAD

**Issue by gyakovlev** [peco/peco#456]
# What is the peco version, OS, architecture? 
N/A

# Abstract

glide.lock refers to spew, but it's not used.
# Discussion

Seems in https://github.com/peco/peco/commit/39dfde1bfeb3455073b4d90b1d552c717dc9c5da
you removed spew, but `glide.lock` still refers to it.
Is it intended or just an artefact?

**Comment by lestrrat** [comment]
spew is used in a debug module.

**Comment by lestrrat** [comment]
Actually, if don't have the build tag enabled, I guess it shouldn't even be there... I guess it's an artifact.

**Comment by gyakovlev** [comment]
@lestrrat 
Can you help me understand if we really need it?
Let me explain a bit,
we are packaging peco for gentoo and wondering if we really need spew as a dependency.

we are not using glide, just vendoring exact same deps using our package manager.

build is started as simple 
```
make build
```

so I guess we don't.

**Comment by lestrrat** [comment]
Yeah, I guess it doesn't need to be there. Too bad glide doesn't support something like "if this build tag is present, include it in the deps".

**Comment by lestrrat** [comment]
lock file is really old anyway. Let me repackage with new libs and what not

**Comment by lestrrat** [comment]
Ugh, I don't know, I'm having glide problems right now and can't really update. Meanwhile, if manually removing spew from your deps work, please go ahead and do that. I'll eventually get around to updating the prereqs :/

**Comment by gyakovlev** [comment]
It totally works for us.
I just wanted to ask you and make sure we are not doing something wrong.
It builds fine for me without spew present at all.

Next time you make a change to build system config we'll see it and adapt accordingly.
Thank you for working on this issue and peco!

**Comment by lestrrat** [comment]
Oops, I forgot to actually remove spew. At least pdebug is fixed, so I should be able to fix it soon

**Comment by lestrrat** [comment]
So `glide` is automatically picking up the dependency from `github.com/lestrrat-go/pdebug`. The dependency only surfaces when you build with the `debug` (or `debug0`) build tag, but glide seems to pick it up regardless. The only solution is to change `github.com/lestrrat-go/pdebug`, but that requires changing the interface... sort of a dilemma here.

Thinking aloud, for now.

**Comment by lestrrat** [comment]
obviously we no longer use glide. closing
